ext/libclementine-remote/remotecontrolmessages.proto

```diff
@@ -367,10 +372,44 @@ message ResponseGlobalSearchStatus {
   optional GlobalSearchStatus status = 3;
 }
 
+// access the FileSystem remotely from a defined root directory
+message RequestListFiles {
+  optional string relative_path = 1;
+}
+
+message FileMetadata {
+  optional string filename = 1;
+  optional bool is_dir = 2;
+}
+
+message ResponseListFiles {
+  enum Error {
+    NONE = 0;
+    ROOT_DIR_NOT_SET = 1;
+    DIR_NOT_ACCESSIBLE = 2;
+    DIR_NOT_EXIST = 3;
+    UNKNOWN = 4;
+  }
+  optional string relative_path = 1;
+  repeated FileMetadata files = 2;
+
+  optional Error error = 3;
+  optional string music_extensions = 4;  // set on server
```

Comment:
If this is really a list it should be `repeated string` rather than encoding that via commas.

Reply:
hum... I've given my opinion above. this field is not really important. I find it more compact to have a coma separated list.
faster to set server side. easy to break down client side.

Reply:
The disadvantage there is that now every piece of code that touches it has to understand how it's encoded (is it comma-separated? is it null-separated, pipe-separated?, etc.).

Reply:
what about a comment in the proto declaration?
if a client would use it, it is normal to check the server encoding choice no?

Reply:
If it's specified as `repeated string` then there's nothing for the client to check really. Either it successfully decodes as `String[]` or it doesn't.

Reply:
alright, then if you don't mind I'm going to move from `ResponseListFiles` to `ResponseClementineInfo`
there is no need to repeat this information.
Or would you prefer another type like `FILES_EXTENSIONS` with a dedicated response message `ResponseFilesExtensions` ?

Reply:
I think it makes sense in `ResponseClementineInfo`.

Reply:
ok cool, I may then rename it in `files_music_extensions`as it is only concerning the browsing of the FileSystem and not the Library for example.